SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -225,11 +225,20 @@ void CFVMFlowSolverBase<V, R>::Allocate(const CConfig& config) {
     }
   }
 
-  /*--- Only initialize when there is a Marker_Fluid_Load defined
-   *--- (this avoids overhead in all other cases while a more permanent structure is being developed) ---*/
-  if ((config.GetnMarker_Fluid_Load() > 0) && (MGLevel == MESH_0)) {
-    Alloc3D(nMarker, nVertex, nDim, VertexTraction);
-    if (config.GetDiscrete_Adjoint()) Alloc3D(nMarker, nVertex, nDim, VertexTractionAdjoint);
+  if (MGLevel == MESH_0) {
+    VertexTraction.resize(nMarker);
+    for (iMarker = 0; iMarker < nMarker; iMarker++) {
+      if (config.GetSolid_Wall(iMarker))
```

Comment:
The idea is to allocate and compute for all solid walls since this is not very expensive.